projects/storefrontlib/src/cms-components/cart/cart-coupon/cart-coupon-anchor/cart-coupon-anchor.service.ts

```diff
@@ -0,0 +1,16 @@
+import { EventEmitter, Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CartCouponAnchorService {
```

Comment:
Why don't you initialize a emitter in component instead of providing it with service?